next.config.js

```diff
@@ -12,7 +12,7 @@ module.exports = {
     HUMAN_INTERNET_URL:
       process.env.HUMAN_INTERNET_URL || "https://human-internet.org/",
     DYNAMIC_LINK_DOMAIN:
-      process.env.DYNAMIC_LINK_DOMAIN || "https://devme.bettersocial.org",
+      process.env.DYNAMIC_LINK_DOMAIN || "https://dev.helio.social",
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change in the default value of `DYNAMIC_LINK_DOMAIN` from `"https://devme.bettersocial.org"` to `"https://dev.helio.social"` seems fine as long as it's intentional and the new domain is correctly configured for your application. Please ensure that all necessary DNS settings, SSL certificates, and any other configurations are properly set up on the new domain. If this domain is used elsewhere in the codebase, make sure to update those instances as well to maintain consistency.
```diff
- process.env.DYNAMIC_LINK_DOMAIN || "https://devme.bettersocial.org",
+ process.env.DYNAMIC_LINK_DOMAIN || "https://dev.helio.social",
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->